_posts/0000-01-02-au97stin.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It is a tragedy that future generations will not get to explore the island of Club Penguin. Curse you Disney!